quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -69,7 +71,7 @@ function startZitiController {
 }
 
 function stopZitiController {
-  killall ziti-controller
+  pkill ziti-controller
```

Comment:
really we need a 'mult-platform way' of doing this better than killall or pkill. Are either of those better/worse for Fedora (and any other *nix variant)? Is there some better way to do this sort of thing in general? capturing a pid on start for example ?

Reply:
Oh dang, you're right. `pkill` is missing in fedora:36. `killall` didn't exist in CentOS 7 is why I changed it. You're right, `kill` is the only truly universal because it's a BASH builtin. It's not hard to add PID handling here. I can do that.